Add unit vary time design documentation and testing details. Contributors: @SGalindo831 and @KobiMurakami

SGalindo831 opened the pull request:
Added our findings and changes to the Time Variance documentation. Also added a separate markdown file that contains the steps on how to create and test the data for Time Variance.

huss commented on unitVaryTime/timeVaryingTesting.md:
````diff
@@ -0,0 +1,115 @@
+delete from cik where source_id = 10 and destination_id = 1; 
+```
+
+Note: In the queries provided above, we have a `` source_id = 10 `` and `` destination_id = 1 ``. The `` source_id `` being used for this is `` Electric_Utility `` and `` destination_id `` being used is `` Electric Utility kWh ``. 
````
I think CIK has the source as you describe but the destination is a graphic unit so it is kWh. Do you agree?

huss reviewed (changes requested):
Thanks to @SGalindo831 for this work. Overall it looks good. I made one small comment to address.

Also, I didn't find that you have signed the [CLA](https://openenergydashboard.org/developer/cla/). It isn't linked to design doc pull requests but is still needed. Let me know if the OED record are incorrect. Also, the description says "our findings and changes". Could you explicitly list the GitHub IDs for everyone involved and make sure they have signed the CLA?

Let me know if this is not clear or you need any help.

SGalindo831 commented:
Ok I pushed the changes. I added the usernames to the documentation, signed the CLA and finally addressed the kWh unit confusion.

huss commented:
@SGalindo831 Thank you for the updates. I think my comments confused you so let me clarify:

1. The GitHub IDs involved go into the PR description and not the actual documents. In general, OED does not put in developer names/ids into the code/documentation. The commits and PRs show who did the work. Thus, can you remove them from documentation (where you recently added them) and put the into the PR description.
2. I still believe you used the destination unit of kWh. You did change the wording but it still seems to have a different unit name. Do you agree I am correct or am I missing something?

Please let me know if anything is not clear.

SGalindo831 commented:
I'm sorry for the confusion. I removed the Github usernames from the documentation. I also corrected the destination_id description to specify "kWh" instead of "Electric Utility kWh". Hopefully this is correct. Thank you for the help.

huss reviewed (approved):
Thanks to @SGalindo831 & @KobiMurakami for this contribution. The updates look good and addressed all the comments. Congratulations on having this work accepted by OED.
